lib/srv/desktop/rdp/rdpclient/src/lib.rs

```diff
@@ -247,12 +247,27 @@ fn connect_rdp_inner(
         KeyboardLayout::US,
         "rdp-rs",
     );
-    // Client for the "rdpdr" channel - smartcard emulation.
+
+    let tdp_sd_acknowledge = Box::new(move |ack: SharedDirectoryAcknowledge| -> RdpResult<()> {
+        unsafe {
```

Comment:
`tdp_sd_acknowledge` is considered `unsafe` because it is declared in an `extern "C"` block

https://github.com/gravitational/teleport/blob/332af8563418474fa93401b12746fbf5cbd9c552/lib/srv/desktop/rdp/rdpclient/src/lib.rs#L798-L805